app/controllers/posts_controller.rb

```diff
@@ -1,13 +1,12 @@
 class PostsController < ApplicationController
   def index
-    # @posts = Post.all
-
-    # render json: @posts
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: @user.id)
   end
 
   def show
-    # @post = Post.find(params[:id])
-
-    # render json: @post
+    @post = Post.find(params[:id])
+    # @user = User.find(params[:user_id])
+    # @post = Post.where(author_id: @user.id).find(params[:id])
```

Comment:
- [OPTIONAL] Kindly make sure to remove the commented code in order to keep your codebase clean and professional.

Reply:
Thanks for your feedback, @levy002